go.mod

```diff
@@ -40,9 +43,12 @@ require (
 	k8s.io/apiserver v0.0.0
 	k8s.io/cloud-provider v0.0.0
 	k8s.io/component-base v0.0.0
+	k8s.io/component-helpers v0.0.0
+	k8s.io/controller-manager v0.0.0
 	k8s.io/klog v1.0.0
 	k8s.io/klog/v2 v2.2.0
-	k8s.io/kubernetes v0.0.0-20201023141757-9e8ad8ce9d8a
+	k8s.io/kubernetes v1.20.0-beta.0.0.20201030204851-4b65f7065284
```

Comment:
Can we try this again with the latest commit of v1.20.0-beta.2? I think https://github.com/kubernetes/kubernetes/pull/96385 needs to be included for things to work as expected.

Reply:
sure, will update!